### PR TITLE
Feature/vonk 4775 fs releasenotes

### PR DESCRIPTION
**Pull request by cknaap**

### CODE REVIEW

File: releasenotes/releasenotes_v5.rst [hunk 1]
```diff
@@ -5,23 +5,91 @@ Current Firely Server release notes (v5.x)
...
+
+  * to the html output (the homepage): ``script nonce="..."``, ``cache-control``, ``content-security-policy``, ``referrer-policy``, ``x-content-type-options``
+  * and to API response: ``cache-control:no-store``
+
+* You can configure limits on Kestrel, see :ref:`hosting_options`, although using a :ref:`reverse proxy<deploy_reverseProxy>` is still preferred.
+* Added a configuration error to the log if the default informationmodel (aka FHIR version) is not loaded in the pipeline.
+
 Fix
 ^^^
+* Composite search parameters are more accurately supported on SQL Server. Previously a match could be made across components, but not anymore. This comes with a database migration, see above.
```
Comment (alexzautke) [inline releasenotes/releasenotes_v5.rst:53]: Can you explain a bit more on detail that the old behavior was wrong and the new behavior is what you actually want?

File: releasenotes/releasenotes_v5.rst [hunk 2]
```diff
@@ -5,23 +5,91 @@ Current Firely Server release notes (v5.x)
...
+
+* The ``HistoryOptions`` are removed as configuration option, so you can remove it from your configuration in ``appsettings.instance.json`` or environment variables as well. The returned resources will be limited by the settings in the ``BundleOptions``, see :ref:`bundle_options`.
+* The Bulk Data Export upgrades (see below) come with a few extra configuration settings, see :ref:`feature_bulkdataexport`
+
+Features
+^^^^^^^^
+* Firely Server is upgraded to the release version (5.0.0) of FHIR R5. If you have your administration database in SQL Server or MongoDB, this means that the conformance resources will be :ref:`re-imported <conformance_import>`.
+* We included ``errataR5.zip`` with fixes to a few resources and search parameters that have errors in the specification. These are imported automatically at startup.
+* We upgraded Firely Server to the latest SDK 5.1.0, see its `releasenotes <https://github.com/FirelyTeam/firely-net-sdk/releases/tag/v5.1.0>`_.
+* Support for §170.315(b)(10) - Electronic Health Information Export, as required by the end of 2023.
```
Comment (alexzautke) [inline releasenotes/releasenotes_v5.rst:32]: I think we should only publish this sentence once we have the official CHPL listing

**Review by alexzautke — CHANGES_REQUESTED** [review]

File: releasenotes/releasenotes_v5.rst [hunk 3]
```diff
@@ -5,23 +5,91 @@ Current Firely Server release notes (v5.x)
...
+* Bulk Data Export is upgraded to BDE 2.0, with support for: //TODO: Check further
```
Comment (eugene-blokhin) [inline releasenotes/releasenotes_v5.rst:33]: //TODO

File: releasenotes/releasenotes_v5.rst [hunk 4]
```diff
@@ -5,23 +5,91 @@ Current Firely Server release notes (v5.x)
...
+  
+  * patient Filter
+  * _elements filter
+  * HTTP POST with a Parameters resource
+  * export to Azure Blob or Azure Files //TODO: Add settings to bulkdateexport.rst (and maybe appsettings.rst) after merge of develop
```
Comment (eugene-blokhin) [inline releasenotes/releasenotes_v5.rst:38]: //TODO

Comment (eugene-blokhin) [reply releasenotes/releasenotes_v5.rst:38]: This has been done, right?

File: releasenotes/releasenotes_v5.rst [hunk 5]
```diff
@@ -5,23 +5,91 @@ Current Firely Server release notes (v5.x)
...
+* AuditEvents generated for interactions with Firely Server using FHIR R5 were missing a link to the Patient compartment in case a Patient resource was created/read/updated/deleted. Now the AuditEvent.patient element is populated in these cases and by this linked to the Patient compartment. Previously generated AuditEvents are therefore not exported as part of a Bulk Data Export request on a Patient level or when using $everything on Patient.
+* Any markdown in the CapabilityStatement is properly escaped.
+* Firely Server does not support the search parameters whose field ``xpathUsage`` (STU3, R4) or ``processingMode`` (R5) is not set to ``normal``. They are now filtered at startup. See :ref:`restful_search_limitations`.
+* ``CapabilityStatement.instantiates`` on the ``<url>/metadata`` endpoint only lists the CapabilityStatements from the administration API that have their ``status:active``.
 * Firely Server did not support to bring a resource that has earlier been deleted back to life with a conditional update while providing the logical id of the resource in the request payload.
```
Comment (eugene-blokhin) [inline releasenotes/releasenotes_v5.rst:74]:
```suggestion
* Firely Server did not support bringing a resource that has earlier been deleted back to life with a conditional update while providing the logical id of the resource in the request payload.
```

File: releasenotes/releasenotes_v5.rst [hunk 6]
```diff
@@ -5,23 +5,91 @@ Current Firely Server release notes (v5.x)
...
+* Sensitive information in the settings that was logged before is now redacted: 
 
+  * the SSL Certificate password
+  * the MongoDB connectionstring
+ 
+* Regarding :ref:`feature_customsp_reindex`: if an erroreneous parameter is provided as ``include``, a proper error is returned.
```
Comment (eugene-blokhin) [inline releasenotes/releasenotes_v5.rst:80]:
```suggestion
* Regarding :ref:`feature_customsp_reindex`: if an erroneous parameter is provided as ``include``, a proper error is returned. 
```

File: releasenotes/releasenotes_v5.rst [hunk 7]
```diff
@@ -5,23 +5,91 @@ Current Firely Server release notes (v5.x)
...
+* ``CapabilityStatement.instantiates`` on the ``<url>/metadata`` endpoint only lists the CapabilityStatements from the administration API that have their ``status:active``.
 * Firely Server did not support to bring a resource that has earlier been deleted back to life with a conditional update while providing the logical id of the resource in the request payload.
+* Sensitive information in the settings that was logged before is now redacted: 
 
+  * the SSL Certificate password
+  * the MongoDB connectionstring
+ 
+* Regarding :ref:`feature_customsp_reindex`: if an erroreneous parameter is provided as ``include``, a proper error is returned. 
+* URL query decoding was revamped. You should not see any differences, but please contact us if you do.
 * Firely Server leniently accepted a literal unescaped "+" sign as part of the request url and didn't interpret it as a reserved character according to `RFC 3986 <https://www.rfc-editor.org/rfc/rfc3986#section-2.2>`_. Firely Server now correctly interprets it as a whitespace.
```
Comment (eugene-blokhin) [inline releasenotes/releasenotes_v5.rst:82]:
```suggestion
* Firely Server leniently accepted a literal unescaped "+" sign as part of the request url and didn't interpret it as a reserved character according to `RFC 3986 <https://www.rfc-editor.org/rfc/rfc3986#section-2.2>`_. Firely Server now correctly interprets it as whitespace.
```

**Review by eugene-blokhin — CHANGES_REQUESTED** [review]

File: releasenotes/releasenotes_v5.rst (hunk 4 above)
Comment (cknaap) [reply releasenotes/releasenotes_v5.rst:38]: Indeed

File: releasenotes/releasenotes_v5.rst (hunk 3 above)
Comment (cknaap) [reply releasenotes/releasenotes_v5.rst:33]: Checked

File: releasenotes/releasenotes_v5.rst (hunk 1 above)
Comment (cknaap) [reply releasenotes/releasenotes_v5.rst:53]: Done.

**Review by eugene-blokhin — APPROVED** [review]

**Review by alexzautke — APPROVED** [review]

**Pull request merged**